Essentials/src/main/java/com/earth2me/essentials/commands/Commandrenamehome.java

```diff
@@ -0,0 +1,96 @@
+package com.earth2me.essentials.commands;
+
+import com.earth2me.essentials.CommandSource;
+import com.earth2me.essentials.User;
+import com.earth2me.essentials.utils.NumberUtil;
+import net.ess3.api.IUser;
+import org.bukkit.Server;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Locale;
+
+import static com.earth2me.essentials.I18n.tl;
+
+public class Commandrenamehome extends EssentialsCommand {
+    public Commandrenamehome() {
+        super("renamehome");
+    }
+
+    @Override
+    public void run(final Server server, final User user, final String commandLabel, final String[] args) throws Exception {
+        User usersHome = user;
+        final String name;
+        final String newName;
+
+        // Allowing both formats /renamehome jroy home1 home | /sethome jroy:home1 home
+        if (args.length == 2) {
+            final String[] nameParts = args[0].split(":", 2);
+            newName = args[1].toLowerCase(Locale.ENGLISH);
+
+            if (nameParts.length == 2) {
+                name = nameParts[1].toLowerCase(Locale.ENGLISH);
+                if (user.isAuthorized("essentials.renamehome.others")) {
+                    usersHome = getPlayer(server, nameParts[0], true, true);
+                    if (usersHome == null) {
+                        throw new PlayerNotFoundException();
+                    }
+                }
+            } else {
+                name = args[0].toLowerCase(Locale.ENGLISH);
+            }
+        } else if (args.length == 3) {
+            if (!user.isAuthorized("essentials.renamehome.others")) {
+                throw new NotEnoughArgumentsException();
+            }
+
+            usersHome = getPlayer(server, args[0], true, true);
+            if (usersHome == null) {
+                throw new PlayerNotFoundException();
+            }
+
+            name = args[1].toLowerCase(Locale.ENGLISH);
+            newName = args[2].toLowerCase(Locale.ENGLISH);
+        } else {
+            throw new NotEnoughArgumentsException();
+        }
+
+        if ("bed".equals(newName) || NumberUtil.isInt(newName) || "bed".equals(name) || NumberUtil.isInt(name)) {
```

Comment:
Is the case-sensitivity here consistent with other home commands? (ie check whether `/home bEd` takes you to your bed or just `/home bed`)

Reply:
Yes, case on bed is ignored on home and delhome